daemons/pacemakerd/pcmkd_subdaemons.c

```diff
@@ -112,6 +112,14 @@ static void pcmk_process_exit(pcmk_child_t * child);
 static gboolean pcmk_shutdown_worker(gpointer user_data);
 static gboolean stop_child(pcmk_child_t * child, int signal);
 
+static bool
+pcmkd_cluster_connected()
+{
+#if SUPPORT_COROSYNC
+    return pcmkd_corosync_connected();
+#endif
```

Comment:
We need a return value for when SUPPORT_COROSYNC is 0

Reply:
Ah, yes we do.  Fixed that and the commit message.

Reply:
Nitpick about the commit message: we actually connect to CPG, not CFG (which is why it's different from checking cfg_handle)

Reply:
And fixed that too.